topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -0,0 +1,312 @@
+>    https://zenodo.org/record/3567224/files/sweet-potato-chloroplast-nanopore-tiny.fastq
+>    ```
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Map the reads to the assembly
+> * Map the Illumina reads (the new "tiny" dataset) to the `polished-assembly.fasta`, the same way we did before, using bwa mem.
+> * This creates one output file: re-name it `illumina-tiny.bam`
+> * Map the Nanopore reads (the new "tiny" dataset) to the `polished-assembly.fasta`. The settings will be the same, except `Select analysis mode` should be `Nanopore`
+> * This creates one output file: re-name it `nanopore-tiny.bam`
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Visualise mapped reads
+> 1. **JBrowse genome browser** {% icon tool %}:
+>    - *"Reference genome to display"*: `Use a genome from history`
+>        - *"Select a reference genome"*: `polished-assembly.fasta`
+>    - *"Produce Standalone Instance"*: `Yes`
+>    - *"Genetic Code"*: `11. The Bacterial, Archaeal and Plant Plastid Code`
+>    - *"JBrowse-in-Galaxy Action"*: `New JBrowse instance`
+>    - *"Insert Track Group"*
+>        - *"Insert Annotation Track"*
+>            - *"Track Type"*: `BAM pileups`
+>            - *"BAM track data"*: `nanopore-tiny.bam`
+>            - *"Autogenerate SNP track"*: `No`
+>            - *Leave the other track features as default*
+>        - *"Insert Annotation Track"*.
+>            - *"Track Type"*: `BAM pileups`
+>            - *"BAM track data"*: `illumina-tiny.bam`
+>            - *"Autogenerate SNP track"*: `No`
+>            - *Leave the other track features as default*
+{: .hands_on}
+
+There is one output file: re-name: `assembly-and-reads`
+* Click on the eye icon to view. (For more room, collapse Galaxy side menus with corner < > signs).
+*  Make sure the bam files are ticked in the left hand panel.
+* Choose a contig in the drop down menu. Zoom in and out with + and - buttons.
+
+![read mapping](../../images/jbrowse-read-mapping-zoomed-out.png "Overview of reads mapped back to the assembly")
+
+
+> ### {% icon question %} Questions
+>
+> 1. What are the differences between the nanopore and the illumina reads?
+> 2. What are some reasons that the read coverage may vary across the reference genome?
+>
+> > ### {% icon solution %} Solutions
+> > 1. Nanopore reads are longer and have a higher error rate.  
+> > 2. There may be lots of reasons for varying read coverage. Some possibilities: In areas of high read coverage: this region may be a collapsed repeat. In areas of low or no coverage: this region may be difficult to sequence; or, this region may be a misassembly.
+> {: .solution}
+{: .question}
+
+# Annotate the assembly
+
+We can now annotate our assembled genome with information about genomic features. A chloroplast genome annotation tool is not yet available in Galaxy. Instead, we can use a web-based tool to annotate (or upload an example annotation).
```

Comment:
It states that there is not an available tool in Galaxy for annotating chloroplastic and mitochondrial genomes, but Prokka can be used for that, according to different publications. It would be interesting to compare the results obtained with Prokka and those obtained with GeSeq.

Reply:
Thank you for this suggestion, I have added it in the text as an option but will keep GeSeq for now as it uses a customised reference dataset and likely provides a better annotation.